src/core/Geometry.js

```diff
@@ -785,8 +785,10 @@ Object.assign( Geometry.prototype, EventDispatcher.prototype, {
 			return;
 
 		}
-
-		mesh.matrixAutoUpdate && mesh.updateMatrix();
+		
+		if ( mesh.matrixAutoUpdate ) {
```

Comment:
Yeah and ChromeOS Apps do not like it, it throws an error. I'm on a chromebook so I've tested it.